Yoast/YoastSEO.js
# Adds passive voice for German.

## Summary

This PR can be summarized in the following changelog entry:
Adds passive voice for German.

## Relevant technical choices:

* Moved the logic to determine the english passive test voice to its own directory. 

## Test instructions

This PR can be tested by following these steps:

For the standalone:
* Set the locale to de_DE
* Add a german text and see it calculates the passive voice. 

In WordPress
* Set the language to german.
* Add a german text and see it calculates the passive voice. 

Fixes #736
